org.diasurgical.DevilutionX.yml

```
@@ -0,0 +1,44 @@
+app-id: org.diasurgical.DevilutionX
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: devilutionx
+rename-desktop-file: devilutionx.desktop
+rename-appdata-file: devilutionx.metainfo.xml
+rename-icon: devilutionx
+
+finish-args:
+    - --socket=fallback-x11
+    - --socket=wayland
+    - --device=dri
+    - --share=network
+    - --share=ipc
+    - --socket=pulseaudio
+modules:
+    - name: devilutionX
+      buildsystem: cmake-ninja
+      config-opts:
+        - -DBUILD_TESTING=OFF
+        - -DCMAKE_BUILD_TYPE=Release
+      sources:
+        - type: archive
+          url: https://github.com/diasurgical/devilutionX/releases/download/1.5.0/devilutionx-src-fully-vendored.tar.xz
+          sha256: 18aee4324c71198a26f15fe2b342552411c2f62039b64cf9b2fa3e249f1bc4f9
+        # CJK fonts
+        - type: file
+          filename: fonts.mpq
+          url: https://github.com/diasurgical/devilutionx-assets/releases/download/v3/fonts.mpq
+          sha256: 6b62e03c42ae4427055e0f292a1beafb5e840c397adaf1f641b909be37b8653e
+          size: 58488019
+        # Diablo shareware demo
+        - type: file
+          filename: spawn.mpq
+          url: https://github.com/diasurgical/devilutionx-assets/releases/download/v3/spawn.mpq
+          sha256: 64427cd7c1ba904eaa2e0031c16a6b136d0ecef9abc888c5ff8344b459356e38
+          size: 25448219
+      post-install:
+        - "sed -i 's/Icon=devilutionx-hellfire/Icon=org.diasurgical.DevilutionX-hellfire/g' ${FLATPAK_DEST}/share/applications/devilutionx-hellfire.desktop"
```

let's try this again

```suggestion
        - "desktop-file-edit --set-icon=org.diasurgical.DevilutionX-hellfire ${FLATPAK_DEST}/share/applications/devilutionx-hellfire.desktop"
```

using `destkop-file-edit` is always prefered.